src/types.jl

```diff
@@ -374,8 +374,7 @@ function FirstOrderPerturbationSolution(retcode, m::PerturbationModel, c::Solver
                                           m.n_x, m.n_y, m.n_p, m.n_ϵ, m.n_z, c.Q, c.η, c.y,
                                           c.x, c.B, D = maybe_diagonal(c.Ω), c.g_x,
                                           A = c.h_x, C = c.C_1,
-                                          x_ergodic = TuringDenseMvNormal(zeros(m.n_x),
-                                                                          c.V), c.Γ)
+                                          x_ergodic = MvNormal(zeros(m.n_x), c.V), c.Γ)
```

Comment:
Construct with the special mean zeros form.  `MvNormal(c.V)` is probably enough.  We will dispatch on this.

Reply:
Yes, there are some types that I missed.